src/core/components/IconButton/IconButton.css

```diff
@@ -0,0 +1,23 @@
+.icon-button--secondary svg,
+path,
+g {
+  fill: var(--secondary-color);
+  stroke: var(--secondary-color);
+  color: var(--secondary-color);
+}
+
+/* .icon-button svg,
+path,
+g { 
+  fill: var(--primary-color-contrast-text);
+  stroke: var(--primary-color-contrast-text);
+  color: var(--primary-color-contrast-text);
+} */
+
+.icon-button--primary svg,
```

Comment:
As we are expecting that the primary style is the default one, then please add here the default class name too:
```css
.icon-button, .icon-button--primary svg path g { ... }
 ```

Reply:
OK, everything added.